library/general/src/lib/ui/greasemonkey.rb

```diff
@@ -86,6 +86,12 @@ def VStackFrames(old)
     end
     module_function :VStackFrames
 
+    # @param old [Yast::Term]
+    # @return    [Yast::Term]
+    # @example
+    #   term(:FrameWithMarginBox, "Title", "arg1", "arg2")
+    #      ->
+    #   Frame("Title", MarginBox(1.45, 0.45, "arg1", "arg2"))
```

Comment:
That is out of scope for this PR. As I say in the module header,
>   These started out in the Expert Partitioner in yast2-storage.
The use case is reusing pieces of this legacy code in the new yast2-partitioner.
 That is why the API and the implementation look old.